qmarkdowntextedit.h

```diff
@@ -102,9 +102,10 @@ class QMarkdownTextEdit : public QPlainTextEdit {
     bool eventFilter(QObject *obj, QEvent *event) override;
     QMargins viewportMargins();
     bool increaseSelectedTextIndention(
-        bool reverse, const QString &indentCharacters = QChar('\t'));
-    bool handleTabEntered(bool reverse,
-                          const QString &indentCharacters = QChar('\t'));
+        bool reverse,
+        const QString &indentCharacters = QChar::fromLatin1('\t'));
```

Comment:
Should be `QLatin1Char('\t')`

Reply:
Doesn't work. You can't construct a QString with a QLatin1Char

Reply:
![image](https://github.com/pbek/qmarkdowntextedit/assets/58736434/ab7c586e-1762-4b6b-a784-efcdf6f27a76)

That's why `QChar::fromLatin1()`

Reply:
It will work if wrapped with a QChar I guess. You can also just use `QStringLiteral()` instead. But either way, doesn't matter much.